examples/agent/affinity/affinity-values.yaml

```diff
@@ -0,0 +1,61 @@
+observe:
+  token:
+    value: YOURS
+  collectionEndpoint:
+    value: https://YOURS.collect.observeinc.com
+
+
+cluster:
+  name: obs-agent-eks-cluster-entity
+  events:
+    pullInterval: 15m
+    enabled: true
+  metrics:
+    enabled: true
+  namespaceOverride:
+    value: k8smonitoring
+
+containers:
+  logs:
+    enabled: true
+  metrics:
+    enabled: false
+
+agent:
+  selfMonitor:
+    enabled: true
+
+config:
+  global:
+    debug:
+      verbosity: normal
+
+deployment-cluster-events:
+  enabled: true
+  namespaceOverride: "k8smonitoring"
+  affinity:
+    nodeAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        nodeSelectorTerms:
+          - matchExpressions:
+              - key: node-type
+                operator: In
+                values: [nouseme]
```

Comment:
@arthur-observe I think you meant to put `[useme]` rather than `[nouseme]`.


```
kubectl label nodes gke-cluster-1-default-pool-0becdf2b-lsnm node-type=useme
kubectl label nodes gke-cluster-1-default-pool-0becdf2b-z7xh node-type=nouseme

helm install affinity-example -n k8smonitoring -f ./affinity-values.yaml ../../../charts/agent

 kubectl get pods -o wide -n k8smonitoring
NAME                                                           READY   STATUS    RESTARTS   AGE   IP           NODE                                       NOMINATED NODE   READINESS GATES
affinity-example-daemonset-logs-metrics-agent-8kvsh            1/1     Running   0          57s   10.16.2.18   gke-cluster-1-default-pool-0becdf2b-g6nq   <none>           <none>
affinity-example-daemonset-logs-metrics-agent-9pjcr            1/1     Running   0          56s   10.16.1.20   gke-cluster-1-default-pool-0becdf2b-z7xh   <none>           <none>
affinity-example-daemonset-logs-metrics-agent-cwpvf            1/1     Running   0          56s   10.16.0.11   gke-cluster-1-default-pool-0becdf2b-lsnm   <none>           <none>
affinity-example-deployment-agent-monitor-b7d76685f-vt6fx      0/1     Pending   0          56s   <none>       <none>                                     <none>           <none>
affinity-example-deployment-cluster-events-6d988b77d7-vzvwv    1/1     Running   0          56s   10.16.1.21   gke-cluster-1-default-pool-0becdf2b-z7xh   <none>           <none>
affinity-example-deployment-cluster-metrics-74fd6d956d-4zg9p   0/1     Pending   0          56s   <none>       <none>                                     <none>           <none>
````

Reply:
`affinity-example-deployment-cluster-events-6d988b77d7-vzvwv` is created in `gke-cluster-1-default-pool-0becdf2b-z7xh` (node-type=nouseme) rather than in `gke-cluster-1-default-pool-0becdf2b-lsnm` (node-type=useme).

Reply:
right again